docs/api/abc.rst

```diff
@@ -0,0 +1,67 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake.abc
+
+.. _disnake_abcs_ref:
+
+Abstract Base Classes
+=====================
+
+This section documents everything under ``disnake.abc.*``.
```

Comment:
Could you add a bit about what ABC's are?

Reply:
It is done in docs/api/index

Reply:
[`37e164b`](https://github.com/DisnakeDev/disnake/pull/722/commits/37e164be0440716bf05833186bb30c117fedef41)